cli/src/virtual_appliance/bootstrap.sh

```diff
@@ -64,6 +66,9 @@ provisioner pillar_set s3clients/s3server/ip \"${DATA_IP}\"
 salt "*" state.apply components.s3clients.config | tee -a "${LOG_FILE}"
 provisioner confstore_export
 
+#reconfigure hare CDF
+salt-call state.apply components.hare.config.prepare
```

Comment:
This should not be necessary on R2.
Check with hare team, which interface they define configures the CDF file.